apps/include/opt.h

```diff
@@ -185,6 +186,8 @@
             "Disallow session resumption on renegotiation"}, \
         {"no_legacy_server_connect", OPT_S_NOLEGACYCONN, '-', \
             "Disallow initial connection to servers that don't support RI"}, \
+        {"ignore_unexpected_eof", OPT_S_IGNORE_UNEXPECTED_EOF, '-', \
+            "Use legacy error handling if there is no close_notify"}, \
```

Comment:
So this needs to be removed, and you need to write actual code in s_client/s_server and call SSL_(CTX_)set_options.

Reply:
Done. Thanks!